reconcile/utils/ocm/manifests.py

```diff
@@ -0,0 +1,39 @@
+from collections.abc import Generator, Mapping
+from dataclasses import dataclass
+from typing import Any
+
+from reconcile.utils.ocm_base_client import OCMBaseClient
+
+
+@dataclass
+class Manifest:
+    def __init__(self, cluster_id: str):
+        self.href = f"/api/clusters_mgmt/v1/clusters/{cluster_id}/external_configuration/manifests"
```

Comment:
Are all API operations under `external_configuration` follow the same convention? If yes, please extract common CRUD methods when adding another one.